string shooter_id
string hit_color_str
---
bool is_elim